modules/client-configuration/pages/contact-methods-saltssh.adoc

```diff
@@ -58,16 +58,16 @@ The Push via Salt SSH feature uses taskomatic to execute scheduled actions using
 The taskomatic job periodically checks for scheduled actions and executes them.
 Unlike Push via SSH on traditional clients, the Push via Salt SSH feature executes a complete [command]``salt-ssh`` call based on the scheduled action.
 
-By default, actions are executed sequentially on each client, and only one action will run at a time.
-You can increase the number of actions that can be executed in parallel, by adding this line to your configuration file, and adjusting the ``number`` upwards.
+By default there's a limit of ``20`` Salt SSH actions that can be executed in parallel.
+You can increase the number of actions that can be executed in parallel, by adding these lines to your configuration file, and adjusting the ``N`` value upwards.
 We recommend you keep the number of parallel actions low, to avoid problems:
 
 ----
-taskomatic.com.redhat.rhn.taskomatic.task.MinionActionExecutor.parallel_threads=N
-org.quartz.threadPool.threadCount=<number>
+taskomatic.com.redhat.rhn.taskomatic.task.SSHMinionActionExecutor.parallel_threads = N
+org.quartz.threadPool.threadCount = <N + 20>
```

Comment:
I do not understand this expression

Reply:
I interpreted it as "increase the current number by 20", but it would be nice to have this expressed a little more clearly.